.ci/azure-pipelines.yml

```
@@ -0,0 +1,42 @@
+name: $(Build.SourceBranchName)-$(Build.SourceVersion)-$(Rev:r)
+
+trigger:
+  - master
+  - refs/tags/*
+
+pr:
+  - master
+
+resources:
+  repositories:
+    - repository: internal-templates
+      type: github
+      name: xamarin/yaml-templates
+      endpoint: xamarin
+    - repository: components
+      type: github
+      name: xamarin/XamarinComponents
+      endpoint: xamarin
+
+jobs:
+  - template: .ci/build.yml@components
+    parameters:
+      publishJob: windows
+      buildType: none
+      areaPath: 'DevDiv\Xamarin SDK'
+      linuxImage: '' # skip linux builds
+      macosImage: '' # skip macos builds
+      validPackagePrefixes:
+        - Microsoft.Azure.Mobile.Client
+
+  - task: MSBuild@1
+    displayName: msbuild LineEditor.sln
+    inputs:
+      solution: LineEditor.sln
+      configuration: Release
+      msbuildArguments: /restore
```

This needs to be under the build job. I also added the Pack task

```suggestion
      steps:
        - task: MSBuild@1
          displayName: msbuild LineEditor.sln
          inputs:
            solution: LineEditor.sln
            configuration: Release
            msbuildArguments: /restore /t:pack /p:PackageOutputPath=$(Build.SourcesDirectory)/output
```